### PR TITLE
add TextOCR dataset converter

### PR DESCRIPTION
**Pull request by gaotongxiao**
TODO: benchmark DBNet and CRNN

### CODE REVIEW

File: tools/data/textrecog/textocr_converter.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    return args
+
+
+def process_img(args, src_image_root, dst_image_root):
+    # Dirty hack for multi-processing
+    img_idx, img_info, anns = args
+    src_img = mmcv.imread(osp.join(src_image_root, img_info['file_name']))
+    labels = []
+    for ann_idx, ann in enumerate(anns):
+        text_label = ann['utf8_string'].lower()
```
Comment (cuhk-hbsun) [inline tools/data/textrecog/textocr_converter.py:30]: Why use `.lower()` for text_label?

File: tools/data/textrecog/textocr_converter.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        raise Exception(
+            f'{annotation_path} not exists, please check and try again.')
+    src_image_root = root_path
+
+    # outputs
+    dst_label_file = osp.join(root_path, dst_label_filename)
+    dst_image_root = osp.join(root_path, dst_image_path)
+    os.makedirs(dst_image_root, exist_ok=True)
+
+    annotation = json.load(open(annotation_path, 'r'))
```
Comment (cuhk-hbsun) [inline tools/data/textrecog/textocr_converter.py:66]:
```suggestion
    annotation = mmcv.load(annotation_path)
```

File: tools/data/textdet/textocr_converter.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+
+def collect_textocr_info(root_path, annotation_filename, print_every=1000):
+
+    annotation_path = osp.join(root_path, annotation_filename)
+    if not osp.exists(annotation_path):
+        raise Exception(
+            f'{annotation_path} not exists, please check and try again.')
+
+    annotation = json.load(open(annotation_path, 'r'))
```
Comment (cuhk-hbsun) [inline tools/data/textdet/textocr_converter.py:24]:
```suggestion
    annotation = mmcv.load(annotation_path)
```

File: tools/data/textrecog/textocr_converter.py (hunk 1 above)
Comment (gaotongxiao) [reply tools/data/textrecog/textocr_converter.py:30]: I followed the implementation in `svt_converter.py` but it seems unnecessary

**Review by cuhk-hbsun — APPROVED** [review]

**Pull request merged**